src/XenobladeChroniclesX/Mods/BattleDamageModGround2/rules.txt

```diff
@@ -36,7 +36,7 @@ $divr = 10
 
 [Preset]
 category = Damage Divisor
-name = Damage / 10
+name = Damage / 50
```

Comment:
the reason I changed it is that its a typo on the original creators part. x10 was labeled as x10 and x50 was labeled as x10 so I fixed that so that x50 is labeled properly